lib/infra_conversion_throttler.rb

```diff
@@ -3,7 +3,7 @@ def self.start_conversions
     pending_conversion_jobs.each do |ems, jobs|
       running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.count }
       $log&.debug("There are currently #{running} conversion hosts running.")
-      slots = (ems.miq_custom_get('Max Transformation Runners') || Settings.transformation.limits.max_concurrent_tasks_per_ems).to_i - running
+      slots = (ems.miq_custom_get('MaxTransformationRunners') || Settings.transformation.limits.max_concurrent_tasks_per_ems).to_i - running
```

Comment:
thnx - this was causing warnings for me

do we have existing code or customers that uses this old variable name?

Reply:
Good question. @fdupont-redhat will this affect existing automate code for customers?

Reply:
Nope. It's used in the throttler only. But the documentation has to be updated to reflect that change.

Reply:
@fdupont-redhat Thanks for the update.